app/views/components/_document-list.html.erb

```diff
@@ -0,0 +1,50 @@
+<%
+  group_index ||= false
+  track_clicks = "data-module=track-click" if group_index
+  dimension28 = items.count.to_s
+%>
+<ol class="app-c-document-list" <%= track_clicks %>>
+  <% items.each_with_index do |link, link_index| %>
+    <li class="app-c-document-list__item">
+      <h3 class="app-c-document-list__item-title">
+        <%
+          data = {}
+          if group_index
+            data = {
+              track_category: 'navDocumentCollectionLinkClicked',
+              track_action: "#{group_index + 1}.#{link_index + 1}",
+              track_label: link[:base_path],
+              track_options: {
+                dimension28: dimension28,
+                dimension29: link[:title]
+              }
+            }
+          end
+        %>
+        <%=
+          link_to(
+            link[:title],
+            link[:base_path],
+            data: data
+          )
+        %>
+      </h3>
+      <ul>
+        <li class="app-c-document-list__attribute">
```

Comment:
To me, it seems weird that the list attributes are part of an unordered list as it doesn't really feel like a list, as it's more a set of specific related attributes/data?